src/main/java/hwr/oop/List.java

```diff
@@ -11,6 +11,7 @@ public class List {
     private String Name;
     private ToDoItem[] ListToDos;
     private String fileName;
+    private String[] Buckets;
```

Comment:
Wollten wir nicht auf Listen umsteigen?

Reply:
würde ich später einarbeiten

Reply:
habe jetzt die Buckets in Listen umgewandelt.